assignments/assignment2/fileresult.txt

```diff
@@ -0,0 +1,2 @@
+Current tool chain : gcc
+./writer: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=7be51cea6f9bdecf25350107f6c7dd226cce0791, for GNU/Linux 3.2.0, not stripped
```

Comment:
@BhaktiRamani it looks like you didn't get the file cross compiled, it's still an x86-64 executable.